Hymns/Infrastructure/Dependency Injection/AppDelegate+Injection.swift

```diff
@@ -28,7 +28,8 @@ extension Resolver: ResolverRegistering {
         registerConverters()
         registerHymnDataStore()
         registerHistoryStore()
-        registerFavoritesStore()
+        registerTagStore()
+        registerFavoriteStore()
```

Comment:
FavoriteStore*

Reply:
? Hmmm isn't that right

Reply:
well the class is "FavoriteStore", so the function should be "registerFavoriteStore" and not "registerFavoritesStore"

Reply:
Are you able to fix this and then submit? Or do you need another approval after that?